src/components/Garden/ModalFlows/ModalFlowBase.tsx

```diff
@@ -14,13 +12,18 @@ import { useActivity } from '@providers/ActivityProvider'
 import { useWallet } from '@providers/Wallet'
 
 import { setAccountSetting } from '@/local-settings'
-
-const indexNumber = {
-  0: 'First',
-  1: 'Second',
-  2: 'Third',
-  3: 'Fourth',
-  4: 'Fifth',
+import { TransactionType } from '@/hooks/constants'
+
+const indexNumber = ['First', 'Second', 'Third', 'Fourth', 'Fifth']
+
+type ModalFlowBaseType = {
```

Comment:
Yah I prefer interface over types too. Interface can be extended and merged if needed. 
More OOP to reuse, refactor and goodie stuffs. 
And Types used to union, interssection stuffs and more closed types.

Exist some excellent article on internet talking a lot about

One reference: https://blog.logrocket.com/types-vs-interfaces-in-typescript/

I think for now its do not do warm keep in Type, but we should know well the difference to know when need to change it :D

Reply:
It's a matter of opinion and choosing. I like better to use Types since i use a lot enum's and that way we have types all the way. Both interface and type can use extends and union. So its comes to your choosing and opinion. Anyone is correct.